cs/configuring.texy

```diff
@@ -94,6 +94,14 @@ application:
 
 `scanDirs` bude nyní obsahovat pouze hodnotu `foo`.
 
+/--json
```

Comment:
Asi by to chtělo vysvětlit, že tohle patří na rozdíl od zbytku do composer.json a jak se to liší od použití RobotLoaderu (sám nevím v čem je to lepší).

Reply:
Je to napsané pod tím, takto to nestačí?
Vůči RobotLoaderu prakticky žádná výhoda není, jen pokud máš presentery ve vendoru, tak je to podstatný detail.

Reply:
Pod tím mi to právě připadá nesrozumitelné.

Reply:
Ony jsou vysvětlivky k direktivám v celé té sekci až pod tím, ale ok.

Reply:
To celkem nevadí, jde o to, že se najednou zcela mění soubor, kterého se to týká, na to je potřeba zřetelně upozornit.